src/components/apps/launch/AppInfo.js

```diff
@@ -105,25 +111,84 @@ const UnavailableMsg = ({ app, hasDeprecatedParams, baseId }) => {
+                }}
+                startIcon={<ArrowBack fontSize="small" />}
+                onClick={() => router.back()}
+            >
+                <Hidden xsDown>{t("back")}</Hidden>
+            </Button>
+            <Button
+                id={buildDebugId(baseId, ids.BUTTONS.DETAILS)}
+                className={classes.detailsButton}
+                onClick={() => setDetailsDrawerOpen(true)}
+                variant={!isMobile ? "outlined" : "text"}
+                size="small"
+                startIcon={<Info color="primary" fontSize="small" />}
+            >
+                <Hidden xsDown>{i18nApps("details")}</Hidden>
+            </Button>
+            <Button
+                id={buildDebugId(baseId, ids.BUTTONS.DOCUMENTATION)}
+                className={classes.detailsButton}
+                onClick={() => setDocDialogOpen(true)}
+                variant={!isMobile ? "outlined" : "text"}
+                size="small"
+                startIcon={<MenuBook color="primary" fontSize="small" />}
+            >
+                <Hidden xsDown>{i18nApps("documentation")}</Hidden>
+            </Button>
             <Typography
                 variant={isMobile ? "subtitle2" : "h6"}
                 className={classes.appInfoTypography}
             >
-                {loadingError ? (
-                    <LoadingErrorDisplay
-                        baseId={baseId}
-                        loadingError={loadingError}
+                <>
+                    {loadingError ? (
+                        <LoadingErrorDisplay
+                            baseId={baseId}
+                            loadingError={loadingError}
+                        />
+                    ) : loading ? (
+                        <Skeleton width={250} />
+                    ) : (
+                        <Typography variant={isMobile ? "subtitle2" : "h6"}>
+                            {app?.name}
+                        </Typography>
+                    )}
+                    <DetailsDrawer
+                        appId={app?.id}
+                        systemId={app?.system_id}
+                        open={detailsDrawerOpen}
+                        onClose={() => setDetailsDrawerOpen(false)}
+                        baseId={buildDebugId(baseId, ids.BUTTONS.DETAILS)}
                     />
-                ) : loading ? (
-                    <Skeleton width={250} />
-                ) : (
-                    app?.name
-                )}
+                    <AppDoc
+                        baseId={buildDebugId(baseId, ids.BUTTONS.DOCUMENTATION)}
+                        open={docDialogOpen}
+                        appId={app?.id}
+                        systemId={app?.system_id}
+                        name={app?.name}
+                        onClose={() => setDocDialogOpen(false)}
+                        isMobile={isMobile}
+                    />
+                </>
             </Typography>
```

Comment:
I still don't think these dialogs need to be wrapped inside a `Typography` (https://github.com/cyverse-de/sonora/pull/188#discussion_r500678979 and https://github.com/cyverse-de/sonora/pull/188#discussion_r500680056).

Reply:
✔️  I wanted to move this out but somehow spaced it.